config/macros.tex

```diff
@@ -387,11 +382,21 @@
 \newcommand{\addressLen}{\ensuremath{\constantstyle{ADDRLEN}}\xspace} % Bit length of an Ethereum address
 \newcommand{\ethWordLen}{\ensuremath{\constantstyle{ETHWORDLEN}}\xspace} % Bit length of an Ethereum word
 \newcommand{\txDefaultGas}{\ensuremath{\constantstyle{DGAS}}\xspace} % Intrinsic gas cost of an Ethereum transaction
+%% The general group and field
+\newcommand{\Curve}{\ensuremath{\curvestyle{Curve}}\xspace} % Chosen curve
+\newcommand{\rCURVE}{\ensuremath{\constantstyle{r}}\xspace} % Prime that defines the scalar field \FFx{\rCURVE} of \Curve
```

Comment:
`\rCURVE` is not a great macros, but is consistent with `\rBN` etc.  `\r` is not available, of course.